sourced/ml/__main__.py

```diff
@@ -192,6 +192,22 @@ def add_parser(name, help_message):
         "input", help="Path to the model file, URL or UUID.")
     dump_parser.add_argument("--gcs", default=None, dest="gcs_bucket",
                              help="GCS bucket to use.")
+    # ------------------------------------------------------------------------
+    merge_df = add_parser("merge_df", "Merge DocumentFrequency models to a singe one.")
```

Comment:
into a single one

Reply:
How about `mdf` instead of `merge_df`? Or `merge-df`

Reply:
ok, 'merge-df'